packages/components/src/input-control/types.ts

```diff
@@ -67,6 +67,23 @@ export type InputChangeCallback<
 	P = {}
 > = ( nextValue: string | undefined, extra: { event: E } & P ) => void;
 
+export type WithTooltipProps = {
+	/**
+	 * React children.
+	 */
+	children: ReactNode;
+	/**
+	 * Whether or not to wrap children in a Tooltip component.
```

Comment:
Probably forgot to update this after copy-pasting

Reply:
Was this comment relating to the `@default false`? I think the description is otherwise relevant to `WithTooltip` as `showTooltip` will determine whether or not the children are wrapped in a tooltip and div.

Reply:
I honestly don't remember the exact reason why I left this comment — the `@default` value was definitely part of the reason, but I don't remember about the rest. Anyway, your reply makes sense, I'll mark this convo and the following one as resolved.